crypto/src/main/scala/org/bitcoins/crypto/CryptoRuntime.scala

```diff
@@ -134,6 +134,8 @@ trait CryptoRuntime {
 
   def publicKey(privateKey: ECPrivateKey): ECPublicKey
 
+  def publicKeyConvert(key: ECPublicKey, compressed: Boolean): ECPublicKey
```

Comment:
Scaladoc please, what is it converting to?

If I understand what this method is meant to do, i would recommend removing the `compressed` parameter and calling it 

```scala
def toCompressedPublicKey(key: ECPublicKey): ECPublicKey
``` 

?